packages/storybook/.storybook/manager.js

```diff
@@ -37,14 +43,39 @@ document.addEventListener('DOMContentLoaded', function() {
             }
             selector.parentNode.insertBefore(element, selector.nextSibling);
             observer.disconnect();
+            releaseSelectorConfig.enabled = false;
```

Comment:
`disconnect` should be enough, you get the observer as second args on the callback function.
```
const callback = function(mutationsList, observer) {
  ...
  observer.disconnect();
}
```